centreon/www/class/centreonUtils.class.php

```diff
@@ -319,17 +319,18 @@ public static function escapeSecure(
         $stringToEscape,
         $escapeMethod = self::ESCAPE_LEGACY_METHOD
     ) {
+        $stringDecoded = htmlspecialchars_decode($stringToEscape);
```

Comment:
What you're doing is dangerous, because you're modifying the behavior of every piece of code that uses this method.
This is not the right place to make the fix

Reply:
Indeed, moved the changes to the template, see the commit below :

`"RowMenu_desc" => htmlspecialchars_decode(CentreonUtils::escapeSecure($host["host_alias"])),
`